h2o-genmodel/src/main/java/hex/genmodel/easy/LabelEncoderColumnMapper.java

```diff
@@ -0,0 +1,11 @@
+package hex.genmodel.easy;
+
+import hex.genmodel.GenModel;
+
+
+public class LabelEncoderColumnMapper extends EnumEncoderColumnMapper {
```

Comment:
why do we need a special class? can you use just EnumEncoderColumnMapper?

Reply:
yes I will